modules/nlu/src/backend/predict-pipeline.ts

```diff
@@ -461,6 +461,7 @@ function MapStepToOutput(step: PredictStep, startTime: number): PredictOutput {
           value: e.value
         },
         meta: {
+          sensitive: e.sensitive,
```

Comment:
I'll take a note to add this line in this [PR](https://github.com/botpress/botpress/pull/3329).